jquery_dom_selector_practice/js/exercise.js

```diff
@@ -85,7 +106,10 @@
  * Bonus: Try to do this with only one jQuery call. $("selector").css({attribute: "value", attribute: "value"});
  */
 
-
+$("#button3").click(question9);
+	function question9(){
+		$('p').css('font-family','Georgia').css('color', 'blue');
```

Comment:
Check out the sample docs here http://api.jquery.com/css/#example-1-3 on lines 21-24. This is what they were going for on this one.